server/chat-plugins/scavengers.ts

```diff
@@ -1775,27 +2062,33 @@ const commands = {
 	 */
 	recycledhunts(target, room, user) {
 		if (!this.can('mute', null, room)) return false;
-		if (room.roomid !== 'scavengers' && !(room.parent && room.parent.roomid === 'scavengers')) return this.errorReply("Scavenger Hunts can only be added to the database in the scavengers room.");
+		if (room.roomid !== 'scavengers' && !(room.parent && room.parent.roomid === 'scavengers')) {
+			return this.errorReply("Scavenger Hunts can only be added to the database in the scavengers room.");
+		}
 
 		let cmd;
 		[cmd, target] = Chat.splitFirst(target, ' ');
 		cmd = toID(cmd);
 
-		if (cmd === '' || cmd === 'help' || !['addhunt', 'list', 'removehunt', 'addhint', 'removehint', 'autostart'].includes(cmd)) {
+		if (
+			cmd === '' ||
+			cmd === 'help' ||
+			!['addhunt', 'list', 'removehunt', 'addhint', 'removehint', 'autostart'].includes(cmd)
+		) {
```

Comment:
```suggestion
		if (!['addhunt', 'list', 'removehunt', 'addhint', 'removehint', 'autostart'].includes(cmd)) {
```

The tests for `''` or `'help'` seem useless here.